docs/source/pyro.generic.rst

```diff
@@ -4,7 +4,7 @@ Generic Interface
 The ``pyro.generic`` module provides an interface to dynamically dispatch Pyro code
 to custom backends.
 
-.. automodule:: pyro.generic
+.. automodule:: pyro.generic.generic
```

Comment:
nit: it would be a little more accurate to name this file `pyro.generic.dispatch`

Reply:
I'm fine with removing this once we finalize the behavior of the `seed` handler in Pyro.

Reply:
Sorry, I misread your earlier comment. My comment above referred to the presence of `generic` module in `_ALIASES` which has since been removed. I'll address this now - I agree `dispatch` is a more accurate name.